### PR TITLE
A bit of code clean up and suggestions from review

### PR DESCRIPTION
**Pull request by theseion**

### CODE REVIEW

File: bitcoinaudit/README.md [hunk 1]
```diff
@@ -9,23 +9,14 @@ block height and the current total BTC supply.
...
 ```
+Other requirements will be installed from [requirements.txt](requirements.txt).
+
 ## Installing
 ```
 python3 -m venv venv  
-
-pip install python-twitter  
-pip install python-bitcoinrpc  
+pip3 install -r requirements.txt
```
Comment (janrothen) [inline bitcoinaudit/README.md:19]: That's how it's done :-)

File: bitcoinaudit/TweetCreator.py [hunk 2]
```diff
@@ -2,35 +2,33 @@
...
+        return """\
+#Bitcoin block {height}
 
 Total supply: {total} BTC
 
 Increase since yesterday:
 +{increase_blocks} blocks
-+{increase_total} BTC"""
\ No newline at end of file
++{increase_total} BTC
```
Comment (janrothen) [inline bitcoinaudit/TweetCreator.py:33]: This would cause a new line in the tweet

File: bitcoinaudit/TwitterClientMock.py [hunk 3]
```diff
@@ -1,10 +1,9 @@
 MOCK_DATA = 'mock_data_twitter_latest_tweet.txt'
 
 class TwitterClient(object):
-	def get_latest_tweet(self):
-		file_object = open(MOCK_DATA, 'r')
-		return file_object.read()
+    def get_latest_tweet(self):
+        with open(MOCK_DATA, 'rb') as file_object:
+            return file_object.read().decode('utf-8')
```
Comment (janrothen) [inline bitcoinaudit/TwitterClientMock.py:6]: Much cleaner

**Review by janrothen — APPROVED** [review]
Thanks!

File: bitcoinaudit/TweetCreator.py (hunk 2 above)
Comment (theseion) [reply bitcoinaudit/TweetCreator.py:33]: Really? According to the spec the first and last empty line should actually be removed. But if you want to be safe, append a backslash to the second to last line. The closing triple quotes should always stand alone on a line.

Comment (janrothen) [reply bitcoinaudit/TweetCreator.py:33]: I've tried it with the """ on the same line before and the tweet was rendered with a blank line at the end.
But I'm not 100% sure :-p

**Pull request merged**